src/chat/ChatScreen.js

```diff
@@ -164,7 +164,7 @@ export default function ChatScreen(props: Props): Node {
           () => undefined,
         );
 
-        if ((content !== undefined && content !== '') || (topic !== undefined && topic !== '')) {
+        if (content !== undefined || topic !== undefined) {
```

Comment:
This change should go in a separate commit that makes clear the reason for the change, as I mentioned in https://github.com/zulip/zulip-mobile/pull/5632#discussion_r1060945522.